vms/avm/vm_benchmark_test.go

```diff
@@ -81,7 +81,7 @@ func GetAllUTXOsBenchmark(b *testing.B, utxoCount int) {
 		utxo := &avax.UTXO{
 			UTXOID: avax.UTXOID{
 				TxID:        ids.GenerateTestID(),
-				OutputIndex: rand.Uint32(),
+				OutputIndex: uint32(randSrc.Int63()),
```

Comment:
Do we still need the `// #nosec G404` above this? It doesn't seem like we do... I would have thought the `nolintlint`er would catch this... But maybe it doesn't check `#nosec` commands...

Reply:
I will look into that and remove that if it's no longer needed.

Reply:
yes, that's true. I've removed that.